src/common/Utilities.py

```diff
@@ -55,9 +55,32 @@ def examples_data_path(data_type):
         return os.path.join(SIRF_INSTALL_PATH , data_path)
     elif SIRF_PATH is not None:
         return os.path.join(SIRF_PATH, 'data', 'examples', data_type)
-    else:
-        errorMsg = 'You need to set the SIRF_DATA_PATH or SIRF_INSTALL_PATH environment variable to allow finding the raw data.'
-        raise ValueError(errorMsg)
+
+    # one final effort
+    try:
+        py_root = sirf.__file__
+        try:
+            # is install dir like $SIRF_PATH/lib/python3.8/site-packages?
+            lib_idx = py_root.split(os.sep).index('lib')
+        except ValueError:
+            # is install dir like $SIRF_PATH/python?
+            lib_idx = py_root.split(os.sep).index('python')
+        potential_install_loc = os.path.join(os.sep, *py_root.split(os.sep)[:lib_idx])
```

Comment:
wrong path on Windows (\ missing after C:)

Reply:
Ah, thanks for checking